squire_lib/src/tournament.rs

```diff
@@ -918,6 +918,59 @@ impl Tournament {
         }
         digest
     }
+
+    /// Generates a round slip in HTML
+    pub fn round_slips_html(self, css: String) -> String {
+        let mut ret: String = "<!DOCTPYE HTML>".to_string();
+
+        ret += "<html lang=\"en\">";
+        ret += "<head><style>";
+        ret += &css;
```

Comment:
For variables, we can still use this:
```suggestion
        let _ = write!(ret, "{css}");
```